@here/harp-omv-datasource/lib/OmvDecoder.ts

```diff
@@ -248,16 +247,13 @@ export class OmvDecoder implements IGeometryProcessor {
             cachedExprResults: new Map()
         };
 
-        const featureId = env.lookup("$id") as number | undefined;
-
         if (this.m_decodedTileEmitter) {
             this.m_decodedTileEmitter.processPointFeature(
                 layer,
                 extents,
                 geometry,
                 context,
-                techniques,
-                featureId
```

Comment:
this featureId was grayed out in VSCode, i.e. it is never used (the featureId is extracted from the context), hence removing it.